### PR TITLE
Update Riot logo

### PR DESCRIPTION
**Pull request by jryans**
2019-09-19 at 11 42

### CODE REVIEW

**Comment by ara4n** [comment]
lgtm, thanks

**Comment by ara4n** [comment]
actually, that logo seems really fuzzy and weirdly resampled... might be better to rasterise down the svg at higher res and then downscale in the browser for poor-man's retina support?

**Comment by jryans** [comment]
Yeah, I just stole the already-resized-by-script icon of matching size from the riot-web repo, so we'd at least have the new colours... anyway, that's why it's fuzzy and such.

Will try for something nicer like you suggest tomorrow.

**Comment by nadonomy** [comment]
@jryans I haven't looked in detail at where the asset is used, but I traced what you checked in here and put these together:

@1x
Slice@1x

@2x
Slice@2x

_(Dropping these here rather than directly checking in as I don't have the repo set up & figured you'd want to sanity check before pushing.)_

**Comment by jryans** [comment]
Okay, updated with Nad's icons. New screenshot posted to the top comment here.

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,2835 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (turt2live) [inline yarn.lock]: this probably doesn't need to be in this PR

Comment (jryans) [reply yarn.lock]: Fair enough, removed! 😁

**Comment by michaelkaye** [comment]
This has been deployed.

**Pull request merged**